ui/src/index.js

```diff
@@ -1,7 +1,26 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
 import './index.css';
 import App from './App';
+import configureStore from './configureStore';
+
+const preloadedState = {
```

Comment:
This is just seed data and will be removed once we're fetching data from the db. Maybe it should live in it's own file though?